### PR TITLE
Feature: SuggestionList styling and renaming

### PR DESCRIPTION
**Pull request by mikipernu**
Rename the Suggestions component into SuggestionList and style the list items as per the designs. 

Additionally: 
- Adds a new icon for comments (dummy for now, i.e. doesn't fetch number of comments yet)
- Refactors tag styling to support styling for specific tags

In addition, some small chores regarding Suggestions themselves and removing a border-line from SuggestionHeader.

screen shot 2018-10-12 at 11 28 26

### CODE REVIEW

File: web/src/views/Index.vue [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <template>
   <div>
     <navigation></navigation>
-    <Suggestions />
+    <suggestion-list></suggestion-list>
```
Comment (hytonevi) [inline web/src/views/Index.vue]: nää voi myös olla ihan <suggestion-list />

File: web/src/components/suggestion/SuggestionContent.vue [hunk 2]
```diff
@@ -1,13 +1,14 @@
 <template>
   <div class="suggestion-content">
 
-    <div>
+    <div v-if="s.tags && s.tags.length > 0">
```
Comment (hytonevi) [inline web/src/components/suggestion/SuggestionContent.vue]: nyt on 5/5

File: web/src/components/suggestion/SuggestionContent.vue [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
       <p><strong>Käsitteen tyyppi</strong></p>
       <p>Maantieteellinen</p>
     </div>
 
     <div v-if="s.preferred_label.fi">
-      <p><strong>Ehdotettu termi suomeksi</strong></p>
+      <p v-if="s.suggestion_type == 'NEW'"><strong>Ehdotettu termi suomeksi</strong></p>
```
Comment (hytonevi) [inline web/src/components/suggestion/SuggestionContent.vue]: vertailuun voit myös käyttää tota suggestionMappings.suggestionType:n arvoja

Comment (mikipernu) [reply web/src/components/suggestion/SuggestionContent.vue]: vaihdetaan seuraavassa implementaatiossa

**Pull request merged**